safe/gui/tools/minimum_needs/needs_profile.py

```diff
@@ -162,6 +162,21 @@ def sort_by_locale(unsorted_profiles, locale):
         profiles = sort_by_locale(profiles, self.locale)
         return profiles
 
+    def precision_of(self, number_as_text):
+        """The number of digits after the decimal will be counted and used
+        as returned as the precision.
+
+        :param: number_as_text: A textual representation of the number whose
```

Comment:
:param number_as_text: A textual representation of the number whose
            precision we wish to determine.
        :type number_as_text: basestring